src/css/reviews.css

```diff
@@ -1 +1,148 @@
-/* reviews styles */
\ No newline at end of file
+/* reviews styles */
```

Comment:
Please use Prettier for CSS files as well. we should have 2 spaces indent and not 4 :)

Reply:
ok

Reply:
ok